backend/controller/user.go

```diff
@@ -0,0 +1,16 @@
+package controller
+
+import (
+	"api/config"
+	"api/models"
+	"encoding/json"
+	"net/http"
+)
+
+func GetAllUsers(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(200)
```

Comment:
Is this pattern good? Will we still be able to send other status codes for errors (ex: if we can't connect to db)

Reply:
This isn't my end goal for error handling, want to do that mainly all at once with perhaps a middleware I will say if we can't connect to the db the service should panic as per config.Connect()

Reply:
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/a3093838-c454-454d-b0d8-6d8f29505a69)

Will be something like that, which will better handle if we have no response and shit, just want to do all middleware at once, won't expand api functionality without middleware though